### PR TITLE
Print instruction for canceling upload

### PR DESCRIPTION
**Pull request by tkf**
I think it'd be user-friendly to mention how to bail out, after printing a slightly unnerving warning.  Now the session would look like

```
To upload a trace, please authenticate, by visiting:

        https://github.com/login/oauth/authorize?client_id=...

You can cancel upload by `Ctrl-C`.
^C
Cancel uploading the trace.
```

### CODE REVIEW

File: src/BugReporting.jl [hunk 1]
```diff
@@ -157,7 +166,18 @@ function upload_rr_trace(trace_directory)
     """)
     println("To upload a trace, please authenticate, by visiting:\n")
     println("\thttps://github.com/login/oauth/authorize?client_id=$GITHUB_APP_ID&state=$(HTTP.escapeuri(connectionId))")
-    s3creds = take!(c)
+    println("\n", "You can cancel upload by `Ctrl-C`.")
+    flush(stdout)
+    s3creds = try
+        take!(c)
+    catch err
+        if err isa InterruptException
```
Comment (tkf) [inline src/BugReporting.jl:174]: Unfortunately, it looks like `InterruptException` is not catchable _sometimes_. See also: https://github.com/JuliaLang/julia/issues/35713

**Comment by iamed2** [comment]
Could there be an environment variable to disable upload? Or maybe a different "type", like `rr-local` or something?

**Comment by tkf** [comment]
I think `--bug-report=justrr` does this:

https://github.com/JuliaLang/BugReporting.jl/blob/d08519c4853255d7e67fefeccc6beeed496d217d/src/BugReporting.jl#L115-L117

We just need to document it (if Keno is OK with making it a stable API).

**Comment by iamed2** [comment]
Ideally we'd see it run `rr_pack` as well. I'd like to be able to generate something we can send to Julia Computing privately.

**Comment by tkf** [comment]
I see.  I think recording without packing is also a nice thing to have for local debugging.  Maybe you can open a PR to add a way to skip the uploading part?  It'd be nice to have some consistent report type names.  Maybe `rr-nopack` (currently `justrr`) and `rr-noupload` or something?

**Pull request merged**